src/CC2Edit/CC2Edit.cpp

```diff
@@ -2537,14 +2537,49 @@ void CC2EditMain::onTestChips2()
         return;
     }
 #ifndef Q_OS_WIN
-    QString winePath = settings.value(QStringLiteral("WineExe")).toString();
-    if (winePath.isEmpty() || !QFile::exists(winePath)) {
-        // Try standard paths
-        winePath = QStandardPaths::findExecutable(QStringLiteral("wine"));
-        if (winePath.isEmpty() || !QFile::exists(winePath)) {
-            QMessageBox::critical(this, tr("Could not find WINE"),
-                    tr("Could not find WINE executable.\n"
-                       "Please configure WINE in the Test Setup dialog."));
+    static auto versionRegex = new QRegExp(QStringLiteral("\\d{10} (?:proton|experimental)-6\\.\\d+-\\d+\n?"));
+    static auto slrSubdir = new QRegExp(QStringLiteral("[a-zA-Z0-9]+_platform_\\d+\\.\\d+\\.\\d+"));
+    QString steamRoot = settings.value(QStringLiteral("SteamRoot")).toString();
+    if (steamRoot.isEmpty() || !QFile::exists(steamRoot)) {
+        // The one which Steam looks for by default
+        steamRoot = QDir::homePath() + QStringLiteral("/.local/share/Steam");

```

Comment:
You could use `QStandardPaths` with `GenericDataLocation` instead of hard-coding `.local/share`